### PR TITLE
Add new Google Play files for Firefox 58 and 59 beta

### PR DESCRIPTION
**Pull request by Delphine**

### CODE REVIEW

File: app/templates/fx_android/release/listing_apr_2016.php [hunk 1]
```diff
@@ -102,8 +102,9 @@
 
 $whatsnew = function ($translations) use ($_) {
     return <<<OUT
-* {$_('A clean, modern visual refresh')}
-* {$_('An all-new home page with quick access to top sites and recent history')}
-* {$_('An experience designed in parallel with the new Firefox Quantum, our fast, modern desktop browser')}
+* {$_('Removed the Firefox Search widget from home screen')}
```
Comment (flodolo) [inline app/templates/fx_android/release/listing_apr_2016.php]: Why did you change the order? This was the last element in Beta.

Comment (flodolo) [reply app/templates/fx_android/release/listing_apr_2016.php]: For reference, this is the current order: https://l10n.mozilla-community.org/stores_l10n/locale/en-US/fx_android/beta/

Comment (Delphine) [reply app/templates/fx_android/release/listing_apr_2016.php]: Yeah I realized why I got it wrong: I was both looking at that *and* the email thread we had with relman - which initally puts that string first. It should be good now just updated it

Comment (flodolo) [reply app/templates/fx_android/release/listing_apr_2016.php]: Yep, order is good now.
